src/services/WebdavClient.ts

```diff
@@ -24,10 +24,18 @@ import { createClient } from 'webdav'
 import { getRootPath, getToken, isPublic } from '../utils/davUtils'
 import { getRequestToken } from '@nextcloud/auth'
 
+const headers = {
+	// Add this so the server knows it is an request from the browser
+	'X-Requested-With': 'XMLHttpRequest',
```

Comment:
Isn't this already added by another bit of code?

Reply:
No, that is the exact issue